task.php

```diff
@@ -0,0 +1,74 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <title>Project Manager - Project Name</title>
+    <script src="https://cdn.tailwindcss.com"></script>
+</head>
+<body class="h-full">
+<header class="p-3 bg-teal-50 flex justify-between">
+    <h1 class="sm:text-5xl text-4xl"><a href="index.php">Project Manager</a></h1>
+    <div class="pr-3 flex">
+        <a href="task.php" class="p-3 bg-slate-300 rounded-l-lg border-y border-l">🇬🇧</a>
+    </div>
+</header>
+<main class="p-3">
+
+<?php
+
+require_once 'src/Services/DatabaseService.php';
+require_once 'src/Models/TasksModel.php';
+require_once 'src/Entities/TaskEntity.php';
+require_once 'src/Services/DeadlineDateService.php';
+require_once 'src/Services/TaskLinkService.php';
+
+$db = DatabaseService::connect();
+$tasksModel = new TasksModel($db);
+
+$displayTask = $tasksModel->selectTaskById($taskIdLink);
+$displayTaskName = $displayTask->name;
+$displayTaskDescription = $displayTask->description;
+$displayTaskEstimate = $displayTask->estimate;
+$displayTaskDeadline = $displayTask->deadline;
```

Comment:
Suggest you don't make variables like this - it's way more code than just putting $displayTask->name into your HTML directly